package.json

```diff
@@ -37,6 +37,8 @@
     "@chakra-ui/react": "^1.8.8",
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
+    "@fontsource/inter": "^4.5.11",
+    "@fontsource/lobster-two": "^4.5.8",
```

Comment:
Give me a reason why using font lib instead of using the font from google font on CSS by using font-face.

Reply:
Sorry, I don't have a particularly good reason. Install with NPM is the first option provided by Chakra UI and also the first option that worked for me. Thinking through now, using font-face would be the better option. Will give font-face a go.

Reply:
font packages have been removed as per discussed.